scripts/x-charts-pro.exports.json

```diff
@@ -430,8 +430,15 @@
   { "name": "useChartRootRef", "kind": "Function" },
   { "name": "useChartsLocalization", "kind": "Variable" },
   { "name": "useDrawingArea", "kind": "Function" },
+  { "name": "useFunnelSeries", "kind": "Function" },
+  { "name": "useFunnelSeriesContext", "kind": "Function" },
```

Comment:
Do we want to export both `useFunnelSeries` and `useFunnelSeriesContext`?

Reply:
Yeah I thought about it before, but decided to release all for consistency.